src/actions/app.js

```diff
@@ -105,6 +106,14 @@ export function setupInitialUrlState(
     try {
       urlState = stateFromLocation(location, profile);
     } catch (e) {
+      if (e.name === 'UrlUpgradeError') {
```

Comment:
I decided to test with the name instead of instanceof, because by experience Errors are weird and I'm more comfortable doing it that way...